### PR TITLE
Sort cleanup

### PR DESCRIPTION
**Pull request by PRabahy**
Allow cycling through the sort order backwards.
Cleanup some redundant while loops.

### CODE REVIEW

File: EvolvioColor/Board.pde [hunk 1]
```diff
@@ -572,6 +572,14 @@ class Board {
       }
     }
   }
+  
+  public void incrementSort() {
+    evoBoard.creatureRankMetric = (evoBoard.creatureRankMetric + 1) % sorts.length;
+  }
+  
+  public void decrementSort() {
+    evoBoard.creatureRankMetric = (evoBoard.creatureRankMetric + sorts.length - 1) % sorts.length;
```
Comment (PRabahy) [inline EvolvioColor/Board.pde]: I have to do the "+ sorts.length + 1". because java's mod function can return negative numbers...

**Comment by mbertheau** [comment]
PR looks good to me. I add you so that you can merge it yourself :)

**Comment by PRabahy** [comment]
@mbertheau Any reason why creating a merge commit is disabled?

**Comment by mbertheau** [comment]
Yes, I find it leads to a confusing history, where the merge commit often contains conflict resolution. I prefer to rebase branches onto the current master, resolve possible conflicts in the branch, and then merge without conflicts.

**Pull request merged**